src/Components/test/E2ETestMigration/Tests/DownloadAnchorTest.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using BasicTestApp;
+using BasicTestApp.RouterTest;
+using Microsoft.AspNetCore.BrowserTesting;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.Testing;
+using Moq;
```

Comment:
Random question, does this test actually use Moq anywhere?

Reply:
Nah, copy-paste shennagins.

Reply:
```suggestion
```